src/themes/greyVest/SearchEditor.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import _ from 'lodash/fp'
+import F from 'futil-js'
+import { observer } from 'mobx-react'
+import { Flex } from '../../'
+import LinkButton from './LinkButton'
+import TreePauseButton from './TreePauseButton'
+import ToggleFiltersButton from './ToggleFiltersButton'
+
+let SearchEditor = ({ mode, setMode, children, QueryBuilder, FiltersBox }) => {
+  let trees = _.flow(
+    React.Children.toArray,
+    _.map(x => x.props || x)
+  )(children)
+  let SearchGroupComponent = mode === 'builder' ? QueryBuilder : FiltersBox
+  let marginTop = { marginTop: 15 }
```

Comment:
Just inline it. It's being used in only one place

Reply:
This was a trick to make it shorter with prettier, because inlining it caused it to be more lines of code 😂. That's a silly reason though, good point.